mkosi/kmod.py

```diff
@@ -180,6 +180,11 @@ def gen_required_kernel_modules(
             mods = set(modulesd.rglob("*.ko*"))
         firmware = set()
 
+    links = {
+        p.resolve().relative_to(root)
```

Comment:
This will resolve the symlink relative to the host root directory which could lead to very wrong and very hard to debug issues depending on the host system.

Please use `chase()` from mkosi.sandbox to resolve the symlinks properly.